### PR TITLE
fix(accordionitem): add handleOpen and handleClose to log when items …

### PR DESCRIPTION
**Pull request by gregferg**
# Purpose of PR

Adds handleOpen and handleClose to `AccordionItem.tsx` so that we can log when the accordionItems are opened and closed.

## PR Checklist

- [x] I have read the relevant `readme.md` file(s)
- [x] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components#commits)
- [x] Tests are added/updated/not required
- [x] Tests are passing
- [x] Storybook stories are added/updated/not required
- [x] Usage notes are added/updated/not required
- [x] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [x] Doesn't contain any sensitive information

### CODE REVIEW

**Review by gui-santos — CHANGES_REQUESTED** [review]
Looks good! Just a suggestion regarding naming the props :)

**Review by m10l — APPROVED** [review]

**Comment by m10l** [comment]
Just a heads up, Travis is failing because of these errors:

```
  61:9   error    'accordionId' is assigned a value but never used                              @typescript-eslint/no-unused-vars
  62:9   error    'expandedClass' is assigned a value but never used                            @typescript-eslint/no-unused-vars
```

File: packages/forma-36-react-components/src/components/Accordion/AccordionItem/AccordionItem.tsx [hunk 1]
```diff
@@ -23,6 +23,15 @@ export interface AccordionItemProps {
...
    */
   testId?: string;
+  /**
+   * A function to be called when the accordian item is opened
+   */
+  handleOpen?: Function;
+  /**
+   * A function to be called when the accordian item is closed
+   */
+  handleClose?: Function;
```
[the remark this answers is not in the document]
Comment (gui-santos) [reply packages/forma-36-react-components/src/components/Accordion/AccordionItem/AccordionItem.tsx]: I'm telling you English is weird

https://www.youtube.com/watch?v=QO178ZfEVME

[the remark this answers is not in the document]
Comment (gregferg) [reply packages/forma-36-react-components/src/components/Accordion/AccordionItem/AccordionItem.tsx]: That it is haha

**Review by gui-santos — APPROVED** [review]
🙌

**Pull request merged**